src/main/java/frc/robot/commands/ShooterHeld.java

```diff
@@ -44,6 +49,7 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+    m_DriveBaseSubsystem.arcadeDrive(0);
```

Comment:
A quick comment on why you set this to zero would be good.  ( To prevent side to side movement while aligning with the limelight. )